openstack/master/main.tf

```diff
@@ -49,6 +49,18 @@ resource "null_resource" "generate-inventory" {
 
 }
 
+# Upload all additional public keys to master node
```

Comment:
Nice! What happens if KubeNow/public_keys does not exist? Did you test this case?